backends/blif/blif.cc

```diff
@@ -117,11 +115,12 @@ struct BlifDumper
 		if (sig.wire->width != 1)
 			str += stringf("[%d]", sig.wire->upto ? sig.wire->start_offset+sig.wire->width-sig.offset-1 : sig.wire->start_offset+sig.offset);
 
-		cstr_buf.push_back(str);
-		return cstr_buf.back().c_str();
+		return str;
 	}
 
-	const char *cstr_init(RTLIL::SigBit sig)
+	#define CSTR(x) (str(x).c_str())
```

Comment:
N.B.: I stuffed this macro in here to avoid rewriting each of the `cstr` calls that were originally below. I'm happy to rewrite them, though, if you'd prefer to not have a macro.

Reply:
If you get rid of the macro I'll be happy to merge this right away.

Reply:
@whitequark all done!